.github/workflows/sync-testnets.yml

```diff
@@ -37,16 +45,34 @@ jobs:
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v2
 
+      - name: Installing requirements
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y make build-essential jq screen lshw dmidecode fio
+
+      - name: Install Docker
+        run: |
+          sudo apt update
+          sudo apt install apt-transport-https ca-certificates curl software-properties-common -y
+          curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+          sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+          sudo apt update
+          sudo apt install docker-ce -y
+
```

Comment:
Not yet decided if it would be running only on ubuntu-latest - now I need to change chiado for sure to self-hosted so better always to have this since it does not break anything and in case of new machine wouldensure that everything is installed